src/utils/hints/tokensHints.js

```diff
@@ -1,4 +1,5 @@
 export const tokensHints = {
+  token: 'Fungible Token is a Smart Contract that implements the AEX-9 Simple Fungible Token Standard. It is used to represent any asset that is fungible, tradeable and transferable.',
```

Comment:
We concluded on using small caps in hints, by the time you had vacation

Reply:
wasn't aware of that but looking at the other hints you are right - fixed